ReactiveCocoa/Swift/ObjectiveCBridging.swift

```diff
@@ -167,7 +167,7 @@ public func toRACSignal<Value: AnyObject, Error: NSError>(signal: Signal<Value?,
                 subscriber.sendError(error)
             case .Completed:
                 subscriber.sendCompleted()
-            default:
+			case .Interrupted:
```

Comment:
In any case, it's always a good idea to not mix indentation changes with other functional changes to make reviewing code easier. I know about the `w=0` flag, but still, it's a good idea to keep the git history clean :)